xbmc/video/windows/GUIWindowVideoNav.h

```diff
@@ -63,4 +64,5 @@ class CGUIWindowVideoNav : public CGUIWindowVideoBase
   virtual IncludeAllSeasonsAndSpecials GetSettingIncludeAllSeasonsAndSpecials();
   virtual int GetFirstUnwatchedItemIndex(bool includeAllSeasons, bool includeSpecials);
   void SelectFirstUnwatched();
+  static XFILE::VIDEODATABASEDIRECTORY::NodeType GetNodeType(const CFileItemList& items);
```

Comment:
This should be an internal function in an unnamed namespace defined in the cpp file. No need for a (static) member function.